lib/.eslintrc

```diff
@@ -148,7 +149,10 @@
     "react/no-unescaped-entities": "off",
     "react/jsx-no-target-blank": "off",
     "react/jsx-fragments": ["error", "syntax"],
-    "react/display-name": "off"
+    "react/display-name": "off",
+    "jsx-a11y/label-has-associated-control": [ 2, {
+      "controlComponents": ["NxTextInput", "NxStatefulTextInput", "NxFilterInput"]
```

Comment:
This rule verifies that `<label>` elements either refer to an `<input>` by id or contain an input.  Since this is static analysis, the plugin can't automatically know whether a given react component within the `<label>` will render an `<input>`.  Here we tell it which components do.